iphone/TitaniumKit/TitaniumKit/Sources/API/TiRootViewController.m

```diff
@@ -1000,7 +1000,16 @@ - (BOOL)shouldRotateToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOr
 - (void)adjustFrameForUpSideDownOrientation:(NSNotification *)notification
 {
   if ((![TiUtils isIPad]) && ([[UIApplication sharedApplication] statusBarOrientation] == UIInterfaceOrientationPortraitUpsideDown)) {
-    CGRect statusBarFrame = [[UIApplication sharedApplication] statusBarFrame];
+    CGRect statusBarFrame = CGRectZero;
+
+    if ([TiUtils isIOSVersionOrGreater:@"13.0"]) {
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
+      statusBarFrame = UIApplication.sharedApplication.keyWindow.windowScene.statusBarManager.statusBarFrame;
```

Comment:
@hansemannn 
If this will get build for iOS SDK < 13.x (e.g build using Xcode 10) and run on iOS 13, status bar frame will always be CGRectZero.
And  I tried your ti.scanner module using test case given inside module, it is not crashing on iOS 13 (GM Seed). Is there any particular test case which is crashing?

Reply:
Wasn't 8.2.0 supposed to be Xcode 11+? If not, I can of course change it. For the scanner:
1. Open the scanner
2. Take a photo
3. Show the taken photo in preview
4. Cancel the photo preview

The app will crash with an uncaught exception.